pyscriptjs/tests/integration/conftest.py

```diff
@@ -162,7 +172,7 @@ def log_message(self, fmt, *args):
     thread = threading.Thread(None, server.run)
     thread.start()
 
-    yield base_url  # Transition to test here
+    yield server, base_url  # Transition to test here
```

Comment:
I know I am the one who suggested it, but it was meant just as a quick hack to move forward, not as the final state.
With the current status we have:
  - a fixture called `dev_sever` which returns a tuple
  - an attribute `PyScriptTest.dev_server` which does *not* contain the fixture, but only the 2nd item

This is very confusing.

A better way to proceed is probably the following:
  - rename the `HTTPServer` class into `DevServer`
  - add an attribute to it called `addr` or `base_url`, which contains what is now in the `base_url` variable
  - then, the `dev_server` fixture can return an instance of `DevServer`, and `PyScriptTest` can just use `dev_server.base_url` to get the address

Reply:
Thanks, done!